Specs/Widgets/NavigationHelpButton/NavigationHelpButtonSpec.js

```diff
@@ -58,8 +60,15 @@ defineSuite([
             expect(widget.viewModel.showInstructions).toEqual(false);
 
             widget.viewModel.showInstructions = true;
-            func(container.firstChild);
-            expect(widget.viewModel.showInstructions).toEqual(true);
+
+// TODO: does anyone have a real fix for this workaround?
```

Comment:
Anyone have a better workaround here or an actual fix if there is one?  Same comment as before.